components/boloforms/sources/common/base.mjs

```diff
@@ -0,0 +1,83 @@
+import { v4 as uuidv4 } from "uuid";
+import boloforms from "../../boloforms.app.mjs";
+import { parseObject } from "../../common/utils.mjs";
+
+export default {
+  props: {
+    boloforms,
+    db: "$.service.db",
+    http: {
+      type: "$.interface.http",
+      customResponse: true,
+    },
+    webhookName: {
+      type: "string",
+      label: "Webhook Name",
+      description: "The name of the webhook to identify on the BoloForms platform.",
+    },
+  },
+  methods: {
+    _setHookId(hookId) {
+      this.db.set("webhookId", hookId);
+    },
+    _getHookId() {
+      return this.db.get("webhookId");
+    },
+    getDocs() {
+      return this.documentId;
```

Comment:
If `withLabel` is true for `documentId`, it will be an object with label & value properties, so you'd need to specify that you want the value.
```suggestion
      return this.documentId.value;
```

Reply:
the endpoint expects exactly an object with value and label, so it's right.